src/ts/components/playbacktogglebutton.ts

```diff
@@ -33,16 +34,31 @@ export class PlaybackToggleButton extends ToggleButton<ToggleButtonConfig> {
         return;
       }
 
-      if (player.isPlaying()) {
+      // always hide the button on the onPlay Event as the user should get feedback on the press of the play button
+      if (player.isPlaying() || (event && event.type === player.EVENT.ON_PLAY)) {
         this.on();
       } else {
         this.off();
       }
     };
 
+    let playbackWarningHandler = (event: WarningEvent) => {
+      if (event && event.code === 5008) {
+        // Warnings[5008] === 'Playback could not be started, user interaction required'
+        // if playback did not start although the buffer was full pause the video (might not have any effect) and display
```

Comment:
Please start sentences with uppercase letters and add punctuation to make it more readable (e.g. `... although the buffer was full, pause ...`)